@pauliescanlon/gatsby-mdx-embed/src/components/Strava/Strava.tsx

```diff
@@ -1,26 +1,27 @@
 import React, { FunctionComponent } from 'react'
 import { GeneralObserver } from '../GeneralObserver'
 export interface IStravaProps {
-  /** The embed URL provided by strava on the activity page */
-  stravaEmbedUrl: string
+  /** The activity ID provided by Strava's share action on the activity page.  Just the text after `https://www.strava.com/activities/`, which should look something like `1234567/embed/abcdefgh` */
```

Comment:
Try to keep these description short as they appear in the Storybook prop table.

Maybe just say "activityId part of the url"... the help docs and code snippets help users work out the rest.